docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner works
+
+## Contrastive metric learning
+
+From an algorithmic perspective,
+**Finetuner** leverages a contrastive metric learning approach to improve your model.
+How does it work?
+
+### Step 1: Convert a model into an Embedding Model
+
+Finetuner interprets the backbone model architecture,
+removes the default *head*, applies *pooling* and freezes layers that do not need to be trained..
+For example, if your pre-trained model is trained for cat-dog classification,
+Finetuner is going to remove the cat-dog classifier and turn your model into an *embedding model*.
+
+This embedding model does not make predictions or outputs a probability,
+but instead outputs a feature vector to represent your data.
+
+### Step 2: Triplet construction and training on-the-fly
+
+Finetuner works on labeled data.
+It expects a `DocumentArray` consisting of `Document`s where each one contains `finetuner_label` corresponding to a class of a specific training example.
```

Comment:
each one contains a `finetuner_label` tag corresponding to the class of a specific ...